tests/dataframe_test.py

```diff
@@ -394,6 +394,19 @@ def test_groupby_standard(self):
         self.assertListEqual(keys[1].to_list(), [333, 222, 111])
         self.assertListEqual(count.to_list(), [1, 2, 3])
 
+        # testing counts with IPv4 column
+        s = ak.DataFrame({"a": ak.IPv4(ak.arange(1, 5))}).groupby("a").count()
+        pds = pd.Series(
+            data=np.ones(4, dtype=np.int64),
+            index=pd.Index(data=np.array(["0.0.0.1", "0.0.0.2", "0.0.0.3", "0.0.0.4"], dtype="<U7")),
+        )
+        self.assertTrue(s.to_pandas().equals(other=pds))
```

Comment:
Just curious as to why this chunk was moved? Not an issue or preventing merge, just curious.

Reply:
Oh i just figured it fit better in `test_groupby_standard` than `test_gb_series` since the `use_series` flag wasn't set. But it's possible I am misunderstanding

Reply:
Fine by me, I was just curious.